goose.go

```diff
@@ -3,17 +3,35 @@ package goose
 import (
 	"database/sql"
 	"fmt"
+	"regexp"
 	"strconv"
 	"sync"
 )
 
+// VerboseLevel verbose level of the goose library
+type VerboseLevel int
+
+const (
+	// VerboseOff disable the log of the executed SQL statements
+	VerboseOff VerboseLevel = iota + 1
+	// VerboseOn log the executed SQL statements
+	VerboseOn
```

Comment:
how many levels do you expect?

Imho, a simple boolean would be enough for tool like goose.

Reply:
Yes, sorry. 
I started with a more complicated idea then decided to switch to this simple solution but I've forgotten to change VerboseLevel type.